src/modules/messaging/index.js

```diff
@@ -75,6 +75,10 @@ export default class Messaging extends ModuleBase {
     return getNativeModule(this).getToken();
   }
 
+  getAPNSToken(): Promise<string> {
```

Comment:
`Promise<string | null>;` if changing to resolve null.

Could we namespace this under `ios`? It's good to keep platform-specific methods outside of the main cross-platform methods.

e.g. this becomes `firebase.messaging().ios.getAPNSToken()`